src/redis-storage.ts

```diff
@@ -53,4 +58,12 @@ export class RedisStorage implements IStorage {
       return;
     }
   }
+
+  async rename(from: string, to: string) {
+    const value = await this.get(getKey(from))
```

Comment:
@angelaraya this `value` doesn't include the original `sessionId`, it gets set based on the key passed to `getSessionFromStorage` so renamed session will be using WebID as its `sessionID`